sklearn/datasets/tests/test_openml.py

```diff
@@ -214,23 +209,11 @@ def test_fetch_openml_iris(monkeypatch):
     expected_missing = 0
 
     _monkey_patch_webbased_functions(monkeypatch, data_id, test_gzip)
-    assert_warns_message(
-        UserWarning,
-        "Multiple active versions of the dataset matching the name"
-        " iris exist. Versions may be fundamentally different, "
-        "returning version 1.",
```

Comment:
Same comment: we still want to check this warning message, unless I am missing something.